package.json

```diff
@@ -32,7 +32,7 @@
     "testnet:compile": "yarn run lint:sol && truffle compile",
     "testnet:deploy": "yarn run lint:sol && truffle deploy --network noritestnet",
     "testnet:migrate": "yarn run lint:sol && truffle migrate --network noritestnet",
-    "test": "yarn run lint:sol && `yarn bin`/truffle test --reset --all",
+    "test": "yarn run lint:sol && truffle test --reset --all",
     "test:gasreport": "cd gas-reporter && truffle test --reset",
```

Comment:
This will prefer the global truffle version if you have one. This can be a tricky thing. I actually need a global truffle version for vscodes Solidity plugin, but don't want to run the global version for command such as this. Thoughts?